Makefile

```diff
@@ -226,7 +231,7 @@ website/docs/README.md: README.md
 	find $(@D)  -type f -name '*.md' | xargs -I{} sed -i 's/\.\/\(.\+\.svg\)/\/img\/\1/g' {}
 	sed -i 's/graphs\//\/img\/graphs\//g' $@
 
-website/build/index.html: website/docs/README.md
+website/build/index.html: website/docs/README.md gen-docs
```

Comment:
Just to be sure, you can run `make website/build/index.html` multiple times and it only builds once, right? In other words, the fact that we have a phony target in the dependencies would not break the dependency tracking because the phony target has no recipe depends entirely on real targets?

Reply:
no it tries to build the website again. So `yarn` is started, but it knows that everything is up to date and terminates quickly

Reply:
The alternative would be to edit the github actions?

Reply:
ok, we can just replace `gen-docs` with `docs/api.md`. I think that works.

Reply:
Nice, this makes sense 👍